src/pages/LandingPages/MGCovid19/MGCovid19.vue

```diff
@@ -162,7 +180,7 @@ export default {
 }
 
 .mg-hero {
-	background-color: $kiva-green;
+	background-color: $white;
```

Comment:
I did notice that when you resize now, the white background shows below the image.

Reply:
Yea the new design updates are all on a white bg so I removed the green in preparation...